src/Extension/ExtensionRegistry.php

```diff
@@ -14,11 +16,40 @@ class ExtensionRegistry
     /** @var array */
     protected $extensionClasses = [];
 
-    public function add(string $extensionClass): void
+    public function addCompilerPass(string $extensionClass): void
     {
         $this->extensionClasses[] = $extensionClass;
     }
 
+    private function addComposerPackages(): void
+    {
+        // We do a try/catch here, instead of using `method_exists`. This is
+        // because PHPStan is being a smart-ass, and takes the state of the
+        // generated `Types` class into account. And that's exactly the point.
+        try {
+            $packages = Types::boltExtension();
```

Comment:
I think you might remove the `try/catch` if you use:

<?php

iterator_to_array(Types::get('bolt-extension'))

Reply:
Ah, yes! much nicer. I've fixed it straight away! 👍